plugins.js

```diff
@@ -0,0 +1,8 @@
+const JFile = require('jfile');
+const {filter} = require('lodash');
+const path = require('path');
+
+module.exports = () => {
+	const pluginFile = new JFile(path.join(__dirname, '/plugins.txt'));
+	return filter(pluginFile.lines);
```

Comment:
what's the purpose of this?

Reply:
This gives us an array of plugins from the plugins.txt file that can be required by the serverless function (loading the .txt file from within the serverless function wasn't working, I think because it's a non-JS file that couldn't be compiled into the built serverless function).